test/CodeGen/AVR/smul-with-overflow.ll

```diff
@@ -0,0 +1,11 @@
+; RUN: llc < %s -march=avr | FileCheck %s
+
+define i1 @signed_multiplication_did_overflow(i8, i8) unnamed_addr {
+; CHECK-LABEL: signed_multiplication_did_overflow:
```

Comment:
Would you be able to add `CHECK` and `CHECK-NEXT` lines to make sure the correct sequence of instructions is generated? Just something basic would suffice.